src/stock-adjustment-mobile/_adjustment-mobile.scss

```diff
@@ -1,8 +1,12 @@
-.adjustment-mobile, .issue-mobile, .receive-mobile {  
+.adjustment-mobile, .issue-mobile, .receive-mobile, .stock-on-hand-mobile {  
     flex: 1 0 auto;
     display: flex;
     flex-direction: column;
 
+    input[type='radio'] {
```

Comment:
Cannot find where this style is used - is it necessary here or maybe shoud be added to shared components is ui-components? Also I think that __stock-on-hand-mobile.scss_ file can be removed since there's no styles added there